PSOperations/RetryOperation.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RetryOperation.swift
+//  PSOperations
+//
+//  Created by Bart Whiteley on 9/22/15.
+//
+
+
+public class RetryOperation: GroupOperation {
+    private let operationProducer: () -> Operation
+    private let shouldRetry: (errors:[NSError], retryCount:Int) -> Bool
+    private var numberOfRetries: Int = 0
+    
+    public convenience init(@autoclosure(escaping) operation: () -> Operation, maximumNumberOfRetries: Int = 5) {
```

Comment:
I would reverse the order of the parameters here, so that you can use a trailing closure if you are also passing in a value for `maxiumumNumberOfRetries`